spec/common-client-libs-sections.json

```diff
@@ -855,10 +858,7 @@
   },
   {
     "title": "Misc",
-    "excludes": [
-      "reference_dart_v1",
-      "reference_dart_v0"
-    ],
+    "excludes": ["reference_dart_v1", "reference_dart_v0"],
```

Comment:
I think we need to add `reference_swift_v1` here as it's currently a dead item in the nav: 
![image](https://user-images.githubusercontent.com/5748289/229860316-662506cb-fee6-4986-9776-735f3e862b40.png)

Reply:
@J0 would you be able to help with this one?